src/app/modules/project/edit/edit.component.html

```diff
@@ -21,11 +21,11 @@
   <div class="offset-2 col-8">
     <h1>Edit project</h1>
 
-    <ng-container *ngIf="projectLoading; else projectLoaded">
+    <ng-container *ngIf="projectLoading">
       <p>Project is being loaded</p>
     </ng-container>
-    <ng-template #projectLoaded>
-      <ng-container *ngIf="project; else noProject">
+    <div [hidden]="projectLoading">
```

Comment:
This syntax has my preference of using here instead of the hidden:
```ts
    <ng-container *ngIf="someBoolean; else elseTemplate">
      
    </ng-container>
    <ng-template #elseTemplate>
      
    </ng-template>
```
The reason being; a `NG if else` ensures the items are not even rendered in the DOM, while hidden still creates the elements in the DOM but just hides them.

Reply:
It was like that before. I changed it because I could not find a way to access the viewChild if it's in an ng-template or ng-container because it's not accessible in ngOnInit. If you have any workaround for this I would love to hear it!